Makefile

```diff
@@ -14,9 +14,7 @@ clean:
 
 .PHONY: build
 build:
-	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
-	mvn package -DskipTests=true
-	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+	mvn compile
```

Comment:
In terms of our `Makefile`s in general we need the `build` target to generate the 'binary' (or in this case the `.jar` file) under Vagrant so that the `start.sh` script called by `ubic` can start the service. That means that developers can run `make build && ubic restart <service>` to test local changes. Otherwise the `package` target would need to be used which will fail locally without a `version` set (and would unnecessarily create a `zip` file under Vagrant also).